queries.sql

```diff
@@ -8,3 +8,44 @@ SELECT name from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* PART 2 */
+BEGIN;
+  UPDATE animals SET species = 'unspecified';
+  SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+  UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+  UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+  DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+  DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+  SAVEPOINT dittoDeleted;
+  UPDATE animals SET weight_kg = weight_kg * (-1);
+  ROLLBACK TO dittoDeleted;
+  UPDATE animals SET weight_kg = weight_kg * (-1) WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+/* How many animals are there? answer: 6 */
+SELECT COUNT(*) FROM animals;
+/* How many animals have never tried to escape? answer: 5 */
+SELECT COUNT(*) FROM animals WHERE escape_attempts>0;
```

Comment:
- [ ] This query selects all animals that **_have never_** escaped not the animals that have escape attempts more than `0`. Kindly fix this.